toketmaster/tests/test_tasa.py

```diff
@@ -0,0 +1,86 @@
+import time
+import unittest
+
+from algosdk.error import IndexerHTTPError
+
+from toketmaster.src import client, models, tasa
+
+
+def wait_to_complete(func, *args, **kwargs):
+    while True:
+        try:
+            return func(*args, **kwargs)
+
+        except IndexerHTTPError:
+            time.sleep(1)
+
+
+class TestTasaIsCreated(unittest.TestCase):
```

Comment:
Test is good but should confirm to the structure of https://github.com/algocertified/sandbox/blob/master/toketmaster/test_create_tasa.py.

Reply:
What do you mean here? What is the structure?